# crossfire wrapper parameters

serial_port: /dev/ttyUSB0

frequency: 120

# a.k.a 300 degree/s
max_roll_rate: 4.363323
max_pitch_rate: 4.363323
max_yaw_rate: 4.363323
max_thrust: 2.0

drone_mass: 0.0846

thrust_k: 376.7
thrust_b: 65
